caffe2/CMakeLists.txt

```diff
@@ -1,41 +1,70 @@
-# Excludes
+# ---[ Excludes
 set(Caffe2_DIR_EXCLUDES
-        "/binaries"
-        "/contrib"
-        "/distributed"
-        "/mpi"
-        "/image"
```

Comment:
They shouldn't be excluded wholesale either. If you can give me a list of OMP-requiring files I can build an explicit exclusion list for them